src/main/java/com/google/api/codegen/util/ruby/RubyNameFormatter.java

```diff
@@ -57,7 +57,7 @@ public String publicFieldName(Name name) {
 
   @Override
   public String varReference(Name name) {
-    return "@" + name.toLowerUnderscore();
+    return name.toLowerUnderscore();
```

Comment:
This change is okay IMO because an `@` variable in Ruby is not a reference -- `@` variables are instance variables.

Also, it's never used by the Ruby transformers.

Reply:
Wait do we need to distinguish between localVarReference and memberVarReference?

Reply:
Good point, I renamed the function to `localVarReference`. Looks like nobody would use a `memberVarReference` method yet, so I haven't added it.